conf/messages.cy

```diff
@@ -581,3 +581,18 @@ appealCoverBothPenalties.p1 = Mae hyn yn eich galluogi i nodi manylion apêl unw
 incompleteSessionData.headingAndTitle = Mae’n ddrwg gennym, mae rhywbeth wedi mynd o’i le
 incompleteSessionData.p1 = Mae problem dechnegol wedi codi. Bydd yn rhaid i chi ddechrau’ch apêl eto. Nid ydym wedi cadw’ch data.
 incompleteSessionData.link = Dychwelyd i’r dudalen gychwyn ar gyfer apeliadau
+
+# Appeal by letter kick out page
+# -----------------------------------------------------------
+appealByLetter.headingAndTitle = You can appeal this penalty by letter!
+appealByLetter.p1 = To ask HMRC to review this penalty, send us:!
+appealByLetter.li.1.1 = details of the steps taken to cancel the VAT registration!
+appealByLetter.li.1.2 = dates of any contact with HMRC on this matter!
+appealByLetter.li.1.3 = any other information you’d like to include to support an appeal!
+appealByLetter.p2 = If HMRC accepts your appeal, we will remove:!
+appealByLetter.li.2.1 = this penalty!
+appealByLetter.li.2.2 = all other VAT late submission or late payment penalties for the same period!
+appealByLetter.inset.1 = Write to:!
+appealByLetter.inset.2 = Solicitor’s Office and Legal Service!
+appealByLetter.inset.3 = HMRC!
+appealByLetter.inset.4 = BX9 1ZT!
```

Comment:
There's a test in the MessagesSpec "the value for each English key must be different to the corresponding value of the Welsh key", so the exclamation marks are there for that test to pass until we get the welsh translations

Reply:
Ah I get you, but that would mean that anyone who turns the Welsh lang toggle on it would display '!' at the end of the line. It's probably best to 'ignore' the test

Reply:
And remove the '!'